src/pages/password-reset.test.tsx

```diff
@@ -15,6 +16,59 @@ describe('Password reset page', () => {
       mockedGetUserProfile.mockRejectedValue({ status: 401 });
     });
 
+    describe('given the email', () => {
+      describe('successfully requests password reset', () => {
```

Comment:
Should it be something like:
```suggestion
      describe('when requesting password reset successfully', () => {
```

Reply:
Updated in [3604bb6](https://github.com/hanam1ni/react-surveyor/pull/55/commits/3604bb6848aa8bbe57d01e967633bb695a7c0359)